### PR TITLE
Update VS Code PowerShell Additional Exe Paths instructions

### PR DESCRIPTION
**Pull request by andschwa**
Unfortunately a breaking change was introduced that we are not reverting, and so these instructions need to be updated.

@sdwheeler @SydneyhSmith

### CODE REVIEW

**Comment by andschwa** [comment]
Resolves https://github.com/PowerShell/vscode-powershell/issues/4167

**Comment by andschwa** [comment]
@sdwheeler I'm not sure about the build warnings, they're on the `[!NOTE]` and `[!TIP]` things, which were already in use and I see in other documentation as well.

File: reference/docs-conceptual/dev-cross-plat/vscode/using-vscode.md [hunk 1]
```diff
@@ -155,59 +155,61 @@ field. For example:
 
 You can add other PowerShell executable paths to the session menu through the
 [Visual Studio Code setting](https://code.visualstudio.com/docs/getstarted/settings):
-`powershell.powerShellAdditionalExePaths`.
+setting `powershell.powerShellAdditionalExePaths`.
```
Comment (sdwheeler) [inline reference/docs-conceptual/dev-cross-plat/vscode/using-vscode.md:158]:
```suggestion
`powershell.powerShellAdditionalExePaths`.
```

Comment (andschwa) [reply reference/docs-conceptual/dev-cross-plat/vscode/using-vscode.md]: Good catch, my eyes went right over that other word.

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**